### ISSUE TITLE
Polylang Support

### ISSUE THREAD

**Issue by spencerfinnell** [Astoundify/wp-job-manager-companies#24]
Slug should be registered as a string that can be translated with Polylang. Then the counts should reflect the current language and the link to the company archive should use the translated slug.

**Comment by welshlamb10** [comment]
@spencerfinnell  - yes definitely needs polylang support. I have a plugin that works that you are welcome to have access to. My problem is the lack of support this plugin has to work with elementor

**Comment by Lens40** [comment]
Yes I have the same issue, switching from WPML to Polylang breaks the languages
